build.gradle

```diff
@@ -70,20 +89,26 @@ jacocoTestReport {
 }
 
 patchPluginXml {
-    changeNotes """
+    changeNotes = """
       <ul>
-        <li></li>
+        <li>[BUGFIX] Fix crashes when starting the IDE.</li>
+        <li>[MAINTENANCE] Update dependencies.</li>
       </ul>
       """
 
-    pluginDescription "Companion plugin for the Ghent University Dodona platform, which allows you to submit exercises right from your favourite JetBrains IDE."
+    pluginDescription = "Companion plugin for the Ghent University Dodona platform, which allows you to submit exercises right from your favourite JetBrains IDE."
```

Comment:
@thepieterdc Can you please add that instructions can be found here: https://docs.dodona.be/en/guides/pycharm-plugin/?

Reply:
Done :)